ios/RNCardscan.m

```diff
@@ -28,13 +28,15 @@ - (void)userDidCancel:(ScanViewController * _Nonnull)scanViewController {
 - (void)userDidScanCard:(ScanViewController * _Nonnull)scanViewController creditCard:(CreditCard * _Nonnull)creditCard {
     [self dismissView];
     NSString *number = creditCard.number;
+    NSString *name = creditCard.name;
     NSString *expiryMonth = creditCard.expiryMonth;
     NSString *expiryYear = creditCard.expiryYear;
 
     self.resolve(@{
         @"action" : @"scanned",
         @"payload": @{
             @"number": number,
+            @"name": name ?: [NSNull null],
```

Comment:
I think this will break android. android currently sets `cardholderName` here: https://github.com/getbouncer/react-native-cardscan/blob/master/android/src/main/java/com/getbouncer/RNCardscanModule.java#L59